oracle-ebs-122/2.1/modules/ROOT/pages/new-reference.adoc

```diff
@@ -0,0 +1,503 @@
+|===
+| Name | Type | Description | Default Value | Required
+| Username a| String | Username for the JMS connection type. |  | x
+| Password a| String | Password for the JMS connection type. |  | x
+| Host a| String | Hostname for the connection type. |  | x
+| Port a| String | Port of the connection type. |  1521 |
+| Database SID a| String | Database system identifier (SID). |  | x
+| Reconnection a| <<Reconnection>> |  Configures a reconnection strategy to use when a connector operation fails to connect to an external server. |  |
+|===
+
+[[jms-config_jms-url]]
+===== JMS URL
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Username a| String | Username for the JMS URL connection type. |  | x
+| Password a| String | Password for the JMS URL connection type. |  | x
+| URL a| String | URL for the connection type. |  | x
+| Reconnection a| <<Reconnection>> |  Configures a reconnection strategy to use when a connector operation fails to connect to an external server. |  |
+|===
+
+== Associated Operations
+* <<invokePlSqlRestService, Invoke PL/SQL REST Service>>
+
+== Associated Sources
+* <<BusinessEventsSource, Business Events Source>>
+
+
+== Operations
+
+[[invokePlSqlRestService]]
+=== Invoke PL/SQL REST Service
+`<oracle-ebs122:invoke-pl-sql-rest-service>`
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Input a| Binary |  |  #[payload] |
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* non-repeatable-stream |  Configures how Mule processes streams. The default is to use repeatable streams. |  |
+| PL/SQL a| String |  PL/SQL name. |  | x
+| Operation a| String |  Operation to execute. |  | x
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation's output. The outcome of the expression is stored in the *Target Variable* field. |  #[payload] |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Binary
+|===
+
+=== For Configurations
+* <<config, Default Configuration>>
+
+==== Throws
+* ORACLE-EBS122:UNKNOWN
+* ORACLE-EBS122:BAD_REQUEST
+* ORACLE-EBS122:METADATA_ERROR
+* ORACLE-EBS122:RETRY_EXHAUSTED
+* ORACLE-EBS122:JMS
+* ORACLE-EBS122:CONNECTIVITY
+* ORACLE-EBS122:UNAUTHORIZED
+* ORACLE-EBS122:INTERNAL_SERVER_ERROR
+* ORACLE-EBS122:REQUEST_FAILED
+* ORACLE-EBS122:FORBIDDEN
+* ORACLE-EBS122:XML_PARSING
+
+
+== Sources
+
+[[BusinessEventsSource]]
+=== Business Events Source
+`<oracle-ebs122:business-events-source>`
+
+[NOTE]
+The *Business Events Source* does not filter the incoming events. The user must filter the events after they arrive to the source. +
```

Comment:
```suggestion
The *Business Events Source* does not filter the incoming events. The user must filter the events after they arrive at the source. +
```

Reply:
You might want to double-check the source name. Not sure if it's actually called *Business Events Source*, or if it's *Business Events* source.

Reply:
It shows in studio as "Business Events Source".